public/gameInfo.php

```diff
@@ -932,13 +1095,46 @@ function submitSetRequest(user, gameID) {
                         echo "</div></div>";
                     }
 
+                    echo "</div>";
+                    echo "<br>";
+                    echo "<div>";
                     $renderRatingControl('Game Rating', 'ratinggame', 'ratinggamelabel', $gameRating[RatingType::Game]);
+                } else {
+                    echo "</div>";
+                    echo "<br><br>";
+                    echo "<div>";
+                }
+
+                // Display claim information
+                if ($user !== null && $flags == $officialFlag && !$isEventGame) {
+                    echo "<div style='float:left;display:inline;width:50%'>";
+
+                    echo "<h4>Claim Information</h4>";
+
+                    $claimExpiration = null;
+                    if ($claimListLength > 0) {
+                        echo "Claimed by: ";
+                        foreach ($claimData as $claim) {
+                            $claimUser = $claim['User'];
+                            $claimExpiration = $claim['Expiration'];
```

Comment:
consider wrapping this in `getNiceDate(strtotime( ... ))`

Reply:
Made suggested change.